pkg/manager/cache.go

```diff
@@ -70,12 +71,14 @@ func NewNamespacedCacheForObject(
 
 	cache, err := ctrlcache.New(mgr.GetConfig(),
 		ctrlcache.Options{
-			Scheme:     mgr.GetScheme(),
-			Mapper:     mgr.GetRESTMapper(),
-			SyncPeriod: resync,
+			Scheme:           mgr.GetScheme(),
+			Mapper:           mgr.GetRESTMapper(),
+			DefaultTransform: ctrlcache.TransformStripManagedFields(),
+			SyncPeriod:       resync,
 			ByObject: map[ctrlclient.Object]ctrlcache.ByObject{
 				object: {
 					Namespaces: GetNamespaceCacheConfigs(namespaces...),
+					Transform:  ctrlcache.TransformStripManagedFields(),
```

Comment:
This one is not required. The cache configs below Namespaces inherits the DefaultTransform from above.

Reply:
@akutz Just fyi, edited ^^ I was looking through the defaulting logic of cache options which is slightly complicated and also testing it.

As far as I can tell DefaultTransform is always used to default Transform on lower levels. Doesn't matter if:
* object is a cluster-wide object
* namespaces is nil
* regular case of a namespaced object that should be watched in >= 1 namespaces